materialized-views/integration_tests/Makefile

```diff
@@ -14,10 +14,10 @@ test-redshift:
 
 test-snowflake:
 	dbt deps
-	dbt seed --target snowflake --full-refresh
-	dbt run --target snowflake --full-refresh --vars 'update: false'
-	dbt run --target snowflake --vars 'update: true'
-	dbt test --target snowflake
+	dbt seed --profile garage-snowflake --full-refresh
```

Comment:
is this intentional? why are we using a different profile here?

Reply:
Oh I see that these aren't for CI, but just for local dev. Hmm I think it's still prefer consistency

Reply:
Whoops... I did not mean to push that change... but I had to switch the Snowflake account I was using for (local) integration tests, because I needed the one with access to enterprise features (: